recipes/catch2/2.x.x/test_package/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ project(test_package)
 
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_EXPORT_COMPILE_COMMANDS YES)
```

Comment:
Initially added for the debugging, but I think it a useful thing so I did not delete it.

Reply:
Please, remove. We don't add debugging stuff to avoid extra storage. You can say it's only one file, but if we start to accept it from all, it may result in some gigabytes in few weeks.